lua/lspconfig/server_configurations/openscad_lsp.lua

```diff
@@ -0,0 +1,32 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'openscad-lsp' },
```

Comment:
One sec, need to verify this cmd (the binary name is correct but it seems to default to TCP transport).

Reply:
Yep, missing `--stdio` flag.

```suggestion
    cmd = { 'openscad-lsp', '--stdio' },
```